Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md

```diff
@@ -0,0 +1,23 @@
+Displays a geographical location in an indicator field using Google Maps.
+
+You need to Setup Google Maps to use this automation in Cortex XSOAR. To view the map, you need to add this automation to the General Purpose Dynamic Section in an indicator layout. When viewing the indicator you can see the location in the indicator field.
```

Comment:
You need to setup Google Maps to use this automation in Cortex XSOAR.To view the map, you will need to edit the indicator layout, add a new "General Purpose Dynamic section" and choose the "ShowLocationOnMap" automation.
After those changes, you can see the location in the edited / indicator layout.